framework/test/mocha/unit/modules/http_api/controllers/delegates.js

```diff
@@ -15,7 +15,7 @@
 'use strict';
 
 const rewire = require('rewire');
-const Bignumber = require('bignumber.js');
+const Bignum = require('@liskhq/bignum');
```

Comment:
This file is the only one using the var name as `Bignum`. All the others use as `BigNum`. I suggest changing it for consistency.